arch/arm/src/common/Toolchain.defs

```diff
@@ -258,6 +262,20 @@ else
   LDFLAGS += -nostdlib
 endif
 
+ifeq ($(CONFIG_ARM_THUMB),y)
+  ZARCH := thumb
+else
+  ZARCH := arm
+endif
+
+ifeq ($(CONFIG_ARCH_FPU),y)
+  ZEABI := gnueabihf
+else
+  ZEABI := gnueabi
+endif
+
+ZIGFLAGS = -target $(ZARCH)-freestanding-$(ZEABI)
```

Comment:
why not use ZFLAGS?

Reply:
For keeping same style with current rust/zig flags.